org.opent2t.sample.hub.superpopular/com.insteon.hub/js/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "opent2t-translator-com-insteon-hub",
+  "version": "1.0.0",
+  "description": "Insteon Hub",
+  "license": "MIT",
+  "main": "thingTranslator.js",
+  "files": [
+    "thingTranslator.js",
+    "package.json",
+	  "manifest.xml",
+	  "common.js"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "githttps://github.com/openT2T/translators.git"
+  },
+  "bugs": {
+    "url": "https://github.com/openT2T/translators/issues"
+  },
+  "homepage": "https://github.com/openT2T/translators#readme",
+  "dependencies": {
+    "request": "^2.74.0",
+    "request-promise": "^4.1.1",
+    "opent2t": "^1.0.1",
+    "opent2t-onboarding-org-opent2t-onboarding-insteon": "^0.0.0",
+    "es6-sleep": "*"
+  },
```

Comment:
Added. Should I add those to other hub translators as well?

---
In reply to: [82904624](https://github.com/openT2T/translators/pull/107#discussion_r82904624) [](ancestors = 82904624)